CHANGES.rst

```diff
@@ -1,4 +1,11 @@
-11.17.22 (Unreleased)
+11.17.23 (2024-06-03)
+=====================
+
+JWST
+----
+- Added several RMAPS and updated selections [#1043]
```

Comment:
I would list out the affected reftypes. will make it easier to track down if something is in question.

Reply:
Done.